### PR TITLE
Improve air infiltration measurement translation, Take 2

### PR DESCRIPTION
**Pull request by bpark1327**
## Pull Request Description
Address additional comments for https://github.com/NREL/hescore-hpxml/pull/190

## Checklist

PR Author: Check these when they're done. Not all may apply. ~~strikethrough~~ and check any that do not apply. 

PR Reviewer: Verify each has been completed.

- [ ] Code changes (must work)
- [ ] Test exercising your feature or bug fix. Check the coverage report in the build artifacts.
- [ ] All other unit tests passing
- [ ] Update translation docs

### CODE REVIEW

File: examples/house1-v2-2.xml [hunk 1]
```diff
@@ -57,6 +63,7 @@
                                     <NominalRValue>13</NominalRValue>
                                 </Layer>
                             </AtticFloorInsulation>
+                            <Area>810</Area>
```
Comment (bpark1327) [inline examples/house1-v2-2.xml:66]: @nmerket When I ran `create_all_example_json.py`, an error was thrown. To resolve the problem, I had to add `Area` element.

**Comment by nmerket** [comment]
Ah yes, good catch. That was left over from the knee wall update I made. Thanks for fixing it.

From: bpark1327 ***@***.***>
Date: Thursday, March 3, 2022 at 2:53 PM
To: NREL/hescore-hpxml ***@***.***>
Cc: Merket, Noel ***@***.***>, Mention ***@***.***>
Subject: Re: [NREL/hescore-hpxml] Improve air infiltration measurement translation, Take 2 (PR #193)

CAUTION: This email originated from outside of NREL. Do not click links or open attachments unless you recognize the sender and know the content is safe.

@bpark1327 commented on this pull request.

________________________________

In examples/house1-v2-2.xml<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNREL%2Fhescore-hpxml%2Fpull%2F193%23discussion_r819092065&data=04%7C01%7Cnoel.merket%40nrel.gov%7C30d2efa8167f4180b9d608d9fd60474c%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637819412237920570%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=%2FGfBKZPEvXw480wxfxKalGRVad4XZKoqLuvK5E6RT7Y%3D&reserved=0>:

> @@ -57,6 +63,7 @@

                                     <NominalRValue>13</NominalRValue>

                                 </Layer>

                             </AtticFloorInsulation>

+                            <Area>810</Area>

@nmerket<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnmerket&data=04%7C01%7Cnoel.merket%40nrel.gov%7C30d2efa8167f4180b9d608d9fd60474c%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637819412237920570%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=1RN9kSzge1B77A5lmvu%2FtxXDBxwvuvdFvGclJCm%2FMCY%3D&reserved=0> When I ran create_all_example_json.py, an error was thrown. To resolve the problem, I had to add Area element.

**Review by nmerket — APPROVED** [review]

**Comment by nmerket** [comment]
@bpark1327 Do we need to create a branch on OS-HEScore to pull in this change?

**Comment by bpark1327** [comment]
> @bpark1327 Do we need to create a branch on OS-HEScore to pull in this change?

I think so. I'll create one shortly and make sure that it doesn't break anything.

**Pull request merged**